### PR TITLE
Update Jamstack Definition

### PR DESCRIPTION
**Pull request by domitriusclark**

### CODE REVIEW

**Comment by philhawksworth** [comment]
screenshot

These two sections are the main call out on this page. If people only read a little, it should be these. So I think they should get a bit of a call out / special visual treatment. Especially the first one. We're suggesting this is THE definition.

<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Edge 102.0.1245.30 on Mac OS 10.15.7
Viewport:  1497 x 1146 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-756--jamstack-site.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Edge&browser_version=102.0&resolution=1497x1146&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-756--jamstack-site.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62841af25a7c550008003e4f&commentID=629f729e23399224130e6abc&resolution=resolved)

**Comment by philhawksworth** [comment]
I'd make these bullet points to make them even easy to digest

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Edge 102.0.1245.30 on Mac OS 10.15.7
Viewport:  1497 x 1146 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-756--jamstack-site.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Edge&browser_version=102.0&resolution=1497x1146&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-756--jamstack-site.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62841af25a7c550008003e4f&commentID=629f7334fcb0b524afceea01&resolution=resolved)

**Comment by philhawksworth** [comment]
While this is true... I think we can remove it. It is covered in the opening definition and the benefit of explaining further doesn't feel like it outweighs the extra noise/info.

 screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Edge 102.0.1245.30 on Mac OS 10.15.7
Viewport:  1497 x 1146 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-756--jamstack-site.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Edge&browser_version=102.0&resolution=1497x1146&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-756--jamstack-site.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62841af25a7c550008003e4f&commentID=629f73ce524b22233fed7b5d&resolution=resolved)

**Comment by philhawksworth** [comment]
I think we can make this page feel more digestible by creating more whitespace between these sections, and also by ensuring generous space between the "Why Jamstack" section and the definitions above.

It might also be worth attempting to make each of these Why Jamstack sections a little more concise. But we can wordsmith that later.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Edge 102.0.1245.30 on Mac OS 10.15.7
Viewport:  1497 x 1146 @2x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-756--jamstack-site.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Edge&browser_version=102.0&resolution=1497x1146&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-756--jamstack-site.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62841af25a7c550008003e4f&commentID=629f74d356a6fa213eecb55e&resolution=resolved)

File: src/css/tailwind.css [hunk 1]
```diff
@@ -18,11 +18,11 @@
     }
   }
   h2 {
-    @apply text-white text-3xl font-bold leading-none mb-8;
+    @apply text-white text-3xl font-bold leading-none mb-2 underline;
```
Comment (philhawksworth) [inline src/css/tailwind.css:21]: Adding the underline makes these headings look like links. I'd avoid that

**Review by philhawksworth — CHANGES_REQUESTED** [review]
I'd like to contribute more to this PR on its way to main... but it's in your own repo rather than in a branch on the jamstack.org repo, which makes it a little tricky.

File: src/site/index.njk [hunk 2]
```diff
@@ -7,85 +7,76 @@ layout: layouts/base.njk
...
-    </p>
-  </a>
+<section class="relative p-10 mb-16">
+  <div class="p-2 rounded-lg bg-gradient-jams" w-full>
+    <div class="flex flex-col bg-blue-900 p-10">
+      <a class="border-b-0" name="what-is-jamstack">
+        <h1>What is <span class="font-extrabold text-transparent bg-clip-text bg-gradient-jams">Jamstack</span>?</h1>
+      </a>
+
+      <p>Jamstack is an architectural approach that decouples the web experience layer from data and business logic, improving flexibility, scalability, performance, and maintainability. Jamstack removes the need for business logic to dictate the web experience. It enables a composable architecture for the web where custom logic and 3rd party services are consumed through well defined APIs.</p>
```
Comment (jlengstorf) [inline src/site/index.njk:22]:
```suggestion
      <p>Jamstack is an architectural approach that decouples the web experience layer from data and business logic, improving flexibility, scalability, performance, and maintainability. Jamstack removes the need for business logic to dictate the web experience. It enables a composable architecture for the web where custom logic and 3rd party services are consumed through APIs.</p>
```

**Review by jlengstorf — APPROVED** [review]
Minor edit for clarity.

**Review by jlengstorf — APPROVED** [review]
Ship it! Thank you!

**Pull request merged**